sdk/digitaltwins/Azure.DigitalTwins.Core/src/RequestOptions.cs

```diff
@@ -1,9 +1,7 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-using System;
-using System.Collections.Generic;
-using System.Text;
+using static System.Net.WebRequestMethods;
```

Comment:
What is this using for?

Reply:
I don't know what you are talking about :D